modules/nw-ovn-k-anp-nbctl.adoc

```diff
@@ -0,0 +1,262 @@
+        protocol: TCP
+        port: 8992
+  - name: "allow-to-open-tenants-and-intranet-and-worker-nodes" # rule3
+    action: "Allow"
+    to:
+    - nodes: # worker-nodes
+        matchExpressions:
+        - key: node-role.kubernetes.io/worker
+          operator: Exists
+    - networks: # intranet
+      - 172.30.0.0/30
+      - 10.0.54.0/19
+      - 10.0.56.38/32
+      - 10.0.69.0/24
+    - namespaces: # open tenants
+        matchLabels:
+          tenant: open
+  - name: "pass-to-restricted-tenants" # rule4
+    action: "Pass"
+    to:
+    - namespaces: # restricted tenants
+        matchLabels:
+          tenant: restricted
+  - name: "default-deny"
+    action: "Deny"
+    to:
+    - networks:
+      - 0.0.0.0/0
+----
+====
+
+.Procedure
+
+. List pods with node information by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-ovn-kubernetes -owide
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                     READY   STATUS    RESTARTS   AGE   IP           NODE                                       NOMINATED NODE   READINESS GATES
+ovnkube-control-plane-5c95487779-8k9fd   2/2     Running   0          34m   10.0.0.5     ci-ln-0tv5gg2-72292-6sjw5-master-0         <none>           <none>
+ovnkube-control-plane-5c95487779-v2xn8   2/2     Running   0          34m   10.0.0.3     ci-ln-0tv5gg2-72292-6sjw5-master-1         <none>           <none>
+ovnkube-node-524dt                       8/8     Running   0          33m   10.0.0.4     ci-ln-0tv5gg2-72292-6sjw5-master-2         <none>           <none>
+ovnkube-node-gbwr9                       8/8     Running   0          24m   10.0.128.4   ci-ln-0tv5gg2-72292-6sjw5-worker-c-s9gqt   <none>           <none>
+ovnkube-node-h4fpx                       8/8     Running   0          33m   10.0.0.5     ci-ln-0tv5gg2-72292-6sjw5-master-0         <none>           <none>
+ovnkube-node-j4hzw                       8/8     Running   0          24m   10.0.128.2   ci-ln-0tv5gg2-72292-6sjw5-worker-a-hzbh5   <none>           <none>
+ovnkube-node-wdhgv                       8/8     Running   0          33m   10.0.0.3     ci-ln-0tv5gg2-72292-6sjw5-master-1         <none>           <none>
+ovnkube-node-wfncn                       8/8     Running   0          24m   10.0.128.3   ci-ln-0tv5gg2-72292-6sjw5-worker-b-5bb7f   <none>           <none>
+----
+
+. Navigate into a pod to look at the northbound database by running the following command:
++
+[source,terminal]
+----
+$ oc rsh -c nbdb -n openshift-ovn-kubernetes ovnkube-node-524dt
+----
++
+. Run the following command to look at the `nbdb.ACL`:
++
+[source,terminal]
+----
+$ ovn-nbctl find ACL 'external_ids{>=}{"k8s.ovn.org/owner-type"=AdminNetworkPolicy,"k8s.ovn.org/name"=cluster-control}'
+----
++
+where::
+cluster-control:: Specifies the name of the `AdminNetworkPolicy` you are troubleshooting.
+AdminNetworkPolicy:: Specifies the type: `AdminNetworkPolicy` or `BaselineAdminNetworkPolicy`.
```

Comment:
Doing another round of review, I see that this element uses example names while the next steps uses placeholders like `<anp_name>`. It would be best to have this consistent.

So it is best to replace `cluster-control` with `<anp_name>` and `AdminNetworkPolicy` with a `<kind>` placeholder.

Reply:
updated